apps/spruce/src/pages/waterfall/types.ts

```diff
@@ -10,13 +9,23 @@ export type WaterfallVersion = {
   version: WaterfallVersionFragment | null;
 };
 
-export type Build = Unpacked<
-  Unpacked<WaterfallQuery["waterfall"]["buildVariants"]>["builds"]
->;
+export type Build = {
+  id: string;
+  tasks: Array<{
+    displayName: string;
+    displayStatusCache: string;
+    execution: number;
+    id: string;
+    status: string;
+  }>;
+  version: string;
+};
 
-export type BuildVariant = Unpacked<
-  WaterfallQuery["waterfall"]["buildVariants"]
->;
+export type BuildVariant = {
```

Comment:
I think we should use these types to type the [`waterfall`](https://github.com/evergreen-ci/ui/blob/ff2db150784ffeba8510a6805dbfdb333ef5fee7/apps/spruce/src/pages/waterfall/useFilters.test.tsx#L424) variable in `useFilters.test.tsx` 👀 I think it would prompt us to delete the `activated`, `displayName`, and `version` fields there